refactor: copilot project (#144)
autokitteh/samples: github_copilot_seats/README.md

---
title: Cancel GitHub Copilot access for inactive users
description: If Copilot was not used in a preceding period by users, unsubscribe and notify them in Slack. Users can ask for their subscription to be reinstated.
integrations: ["githubcopilot", "slack"]
categories: ["DevOps"]
---

# GitHub Copilot Seat Pruning

> This automation enumerates once a day all the users in the GitHub organization
> that have access to [Copilot](https://github.com/features/copilot). If any of
> them haven't used it in a preceding period of time, it automatically marks
> their seat for cancellation in the next billing cycle, and notifies them in a
> Slack DM.

Reviewer: ```suggestion
This automation runs daily to list all the users in the GitHub organization who have access to [Copilot](https://github.com/features/copilot). If any of
them haven't used it in a preceding period of time, it marks
their seat for cancellation in the next billing cycle, and notifies them in a
Slack DM.
```

Reviewer: I think "enumerates" makes it unnecessarily difficult to understand. And I took out "automatically" because I think it's redundant.
